docs/hazmat/backends/interfaces.rst

```diff
@@ -599,7 +601,9 @@ A specific ``backend`` may provide one or more of these interfaces.
         :return: A new instance of
             :class:`~cryptography.hazmat.primitives.asymmetric.dh.DHPrivateKey`.
 
-    .. method:: generate_dh_private_key_and_parameters(self, key_size)
+    .. method:: generate_dh_private_key_and_parameters(self, generator, key_size)
```

Comment:
This should remove `self` (I realize it was there before, but we should fix it now!)